src/main/kotlin/model/IM.kt

```diff
@@ -1,33 +1,48 @@
 package uk.gov.justice.hmpps.architecture
 
 import com.structurizr.model.Model
+import com.structurizr.model.Person
 import com.structurizr.model.SoftwareSystem
 import com.structurizr.view.AutomaticLayout
 import com.structurizr.view.ViewSet
 
 class IM private constructor() {
   companion object : HMPPSSoftwareSystem {
     lateinit var system: SoftwareSystem
+    lateinit var i2nTeam: Person
 
     override fun defineModelEntities(model: Model) {
       system = model.addSoftwareSystem(
         "IM",
-        "Interventions Manager\nHolds records of interventions delivered to services users in the community"
+        "(Interventions Manager) Schedules appointments and records service user progress on accredited programmes"
       ).apply {
         ProblemArea.GETTING_THE_RIGHT_REHABILITATION.addTo(this)
       }
+
+      i2nTeam = model.addPerson(
+        "i2n",
+        "Provides and maintains the Intervention Manager service"
+      ).apply {
+        addProperty("previous-name", "Northgate")
+        Tags.PROVIDER.addTo(this)
+      }
     }
 
     override fun defineRelationships() {
-      Delius.system.uses(system, "pushes active sentence requirements or licence conditions which are of interest to IM to", "IAPS")
-      InterventionTeams.interventionServicesTeam.uses(system, "create new interventions in")
-      InterventionTeams.npsProgrammeManager.uses(system, "create new interventions in")
-      InterventionTeams.crcTreatmentManager.uses(system, "create new interventions in")
+      Delius.system.uses(system, "pushes active sentence requirements or licence conditions to", "IAPS")
+      InterventionTeams.npsProgrammeManager.uses(system, "schedules accredited programme appointments and tracks service user attendance in")
+      InterventionTeams.crcTreatmentManager.uses(system, "schedules accredited programme appointments and tracks service user attendance in")
```

Comment:
Call both teams "treatment manager"s (people managing and delivering interventions)

Reply:
Done in 5d972c13c8c0c8c1de7816184e5905c4928f48c0